DQMOffline/Trigger/plugins/JetMETMonitor.cc

```diff
@@ -0,0 +1,1237 @@
+        jetmetPhiME_.numerator -> Fill(met_phi[i]);
+        jetmetVsLS_.numerator -> Fill(ls, met_pt[i]);
+     }
+  }
+  if (isJetFrac) {
+     // Jet Respnse plot //  hlt / offline jet 
+     // offline Jet // 
+     edm::Handle<reco::PFJetCollection> jetHandle;
+     iEvent.getByToken( pfjetToken_, jetHandle );
+     std::vector<reco::PFJet> jets;
+     if ( int(jetHandle->size()) < njets_ ) return;
+     for ( auto const & jet : *jetHandle ) {
+       jets.push_back(jet);
+     }
+     if ( int(jets.size()) < njets_ ) return;
+     // HLT Jet Selection
+     edm::Handle<reco::PFJetCollection> hltjetHandle;
+     iEvent.getByToken( hltpfjetToken_, hltjetHandle );
+     std::vector<reco::PFJet> hlt_jets;
+     if ( int(hltjetHandle->size()) < njets_ ) return;
+     for ( auto const & jet : *hltjetHandle ) {
+        if (jet.pt() < ptcut_) {continue;}
+        hlt_jets.push_back(jet);
+     }
+     if ( int(hlt_jets.size()) < njets_ ) return;
+
+     std::vector <float> jet_pt;
+     std::vector <float> jet_eta;
+     std::vector <float> jet_phi;
+     std::vector <float> jet_dR;
+     jet_pt.clear();
+     jet_eta.clear();
+     jet_phi.clear();
+     jet_dR.clear();
+     for (unsigned int i = 0; i < hlt_jets.size(); ++i)
+     {
+        hltjetJetFracME_NHEF.Hist->Fill(hlt_jets[i].neutralHadronEnergyFraction());
+        hltjetJetFracME_NEEF.Hist->Fill(hlt_jets[i].neutralEmEnergyFraction());
+        hltjetJetFracME_CHEF.Hist->Fill(hlt_jets[i].chargedHadronEnergyFraction());
+        hltjetJetFracME_MuEF.Hist->Fill(hlt_jets[i].muonEnergyFraction());
+        hltjetJetFracME_CEEF.Hist->Fill(hlt_jets[i].chargedEmEnergyFraction());
+        hltjetJetFracME_NHEFVshltjetPt.Hist->Fill(hlt_jets[i].pt(),hlt_jets[i].neutralHadronEnergyFraction());
+        hltjetJetFracME_NEEFVshltjetPt.Hist->Fill(hlt_jets[i].pt(),hlt_jets[i].neutralEmEnergyFraction());
+        hltjetJetFracME_CHEFVshltjetPt.Hist->Fill(hlt_jets[i].pt(),hlt_jets[i].chargedHadronEnergyFraction());
+        hltjetJetFracME_MuEFVshltjetPt.Hist->Fill(hlt_jets[i].pt(),hlt_jets[i].muonEnergyFraction());
+        hltjetJetFracME_CEEFVshltjetPt.Hist->Fill(hlt_jets[i].pt(),hlt_jets[i].chargedEmEnergyFraction());
+     }
+  }// end of isJetFrac
+
+}
+
+void JetMETMonitor::fillHistoPSetDescription(edm::ParameterSetDescription & pset)
+{
+  pset.add<int>   ( "nbins");
+  pset.add<double>( "xmin" );
+  pset.add<double>( "xmax" );
+}
+
+void JetMETMonitor::fillHistoLSPSetDescription(edm::ParameterSetDescription & pset)
+{
+  pset.add<int>   ( "nbins", 2500);
+}
+
+void JetMETMonitor::fillDescriptions(edm::ConfigurationDescriptions & descriptions)
+{
+  edm::ParameterSetDescription desc;
+  desc.add<std::string>  ( "FolderName", "HLT/JetMETMonitor" );
+
+  desc.add<edm::InputTag>( "met",      edm::InputTag("pfMet") );
+  desc.add<edm::InputTag>( "pfjets",     edm::InputTag("ak4PFJetsCHS") );
+  desc.add<edm::InputTag>( "hltpfjets",     edm::InputTag("hltAK4PFJets") ); // ak4jet about HLT 
```

Comment:
where this collection come from ?
I doubt it is available in RAW format which is the standard format processed @tier0 in the PromptReco
I'm really confused :(

or are you planning to make use of this in the HLTMonitor PD ?
it was not defined in the google doc in this way ....